src/components/labels/_labels.scss

```diff
@@ -40,6 +40,10 @@ $includeHtml: false !default;
       flex-shrink: 0;
       align-items: center;
       margin-right: gutter(0.25);
+      
+      // To fix alignment issue caused by Proxima font
+      position: relative;
```

Comment:
@seahindeniz 👍 but maybe we should actually add this fix to text in label, as it's Proxima issue, so it's related to text, not to icon? WDYT?

Reply:
Oops, sorry. I thought I was added into text. Text @burakukula